### PR TITLE
Lose95

### PR DESCRIPTION
**Pull request by slobters**
# New Port for Lose95

## Game Information
- **Title**: Lose95
- **URL**: [{ Link to project page or source control page }https://kultisti.itch.io/lose95](https://kultisti.itch.io/lose95)

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/lose95/Lose95.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+TOOLDIR="$GAMEDIR/tools"
```
Comment (Slayer366) [inline ports/lose95/Lose95.sh:16]:
```suggestion
```
Have 2 of these

**Review by Slayer366 — APPROVED** [review]

File: ports/lose95/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## Notes
+
+A special thank you to [Kultisti](https://kultisti.itch.io/) for making this great PC GUI action game and allowing us to distribute it Ready to run !
```
Comment (Cebion) [inline ports/lose95/README.md:3]: Missing URL to the game

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/lose95/README.md (hunk 2 above)
Comment (slobters) [reply ports/lose95/README.md]: Added link to the Lose95 game on Itch instead of the creator's Itch page.

**Review by Cebion — APPROVED** [review]

**Review by Slayer366 — APPROVED** [review]

File: ports/lose95/Lose95.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+$ESUDO chmod 777 "$GAMEDIR/gmloadernext"
+
+# Splash
+[ "$CFW_NAME" == "muOS" ] && splash "splash.png" 1 # workaround for muOS
+$ESUDO splash "splash.png" 10000 &
+
+$GPTOKEYB "gmloadernext" -c "controls.gptk" &
+pm_platform_helper "$GAMEDIR/gmloadernext"
+$ESUDO chmod +x "$GAMEDIR/gmloadernext"
```
Comment (cdeletre) [inline ports/lose95/Lose95.sh:42]: This is already done in line 34 :)

**Pull request merged**